src/routes/Tool/Interests.tsx

```diff
@@ -195,6 +217,49 @@ const VirtualAssistant = ({
           />
         </div>
       </div>
+      {selectedKiinnostuksetViewVisible && (
+        <div className="absolute w-full mt-6 h-full left-0 z-21">
+          <div className="bg-white rounded shadow-[0_-1px_24px_rgba(0,0,0,0.25)] h-full flex flex-col">
```

Comment:
```tsx
shadow-[0_-1px_24px_rgba(0,0,0,0.25)]
```
If wanted, this could be new shadow in design-system also, but did not yet add this if this is the only place for it.

```
--shadow-panel: 0px -1px 24px 0px rgba(0, 0, 0, 0.25);
```